app/models/settings.py

```diff
@@ -44,6 +44,8 @@ def __init__(self) -> None:
         self._todds_active_mods_target: bool = False
         self._todds_dry_run: bool = False
         self._todds_overwrite: bool = False
+        self._enable_themes: bool = False
```

Comment:
Why have the defaults here be different from that in apply default settings when you are just gonna set it in apply default settings right after?

Reply:
done